lms/static/js/spec/verify_student/make_payment_step_view_spec.js

```diff
@@ -216,6 +217,13 @@ define([
                     'Try the transaction again in a few minutes.'
                 );
             });
+            it( 'check Initialize method without AB testing ', function() {
+                var view = createView();
+                expect( view.templateName ).toEqual("make_payment_step");
+                expect( view.btnClass ).toEqual("action-primary");
```

Comment:
For consistency please continue to use the quotation mark convention of the file you are working on (i.e. these should be ' instead of ")